### PR TITLE
SW-3611 Change the object height section on the working area page

### PR DESCRIPTION
**Pull request by khaledsherkawi**
SW-3611 Rename 'Object Height' into 'Working Height' + modify text

### CODE REVIEW

File: octoprint_mrbeam/static/js/app/view-models/working-area.js [hunk 1]
```diff
@@ -626,7 +626,7 @@ $(function () {
                     duration_processing: Date.now() - start_ts,
                     gco_generator_info: {
                         generator: beamos_vers ? "beamOS" : null,
-                        version: beamos_vers ? beamos_vers : null,
+                        version: beamos_vers,
```
Comment (JFSolution) [inline octoprint_mrbeam/static/js/app/view-models/working-area.js:629]: why you removed this?

File: octoprint_mrbeam/templates/tab_workingarea.jinja2 [hunk 2]
```diff
@@ -129,51 +129,58 @@
...
+                                            </div>
 
                                             <div class="accordion-body collapse in" id="wa_pane_object_height_body">
-												<div class="accordion-inner">
+                                                <div class="accordion-inner">
                                                     <div>
-														<span class="muted">{{ _('Set the height of the surface above the bottom for the camera preview to be accurate:') }}</span>
+														<span class="muted">{{ _('Set the height at which you want to laser so that the camera preview is accurate (including cutting aids or the Mr Beam Height Extension). For more information on the working-height, see ') }}<a
+                                                                target="_blank"
+                                                                href="{{ _("https://support.mr-beam.org/en") }}/support/solutions/articles/43000708214-how-to-adjusting-the-working-height-correctly">
```
Comment (JFSolution) [inline octoprint_mrbeam/templates/tab_workingarea.jinja2]: pleas add google analytics keywords

File: octoprint_mrbeam/camera/corners.py
Comment (JFSolution) [inline octoprint_mrbeam/camera/corners.py:1]: why you needed to refactor this?

**Review by JFSolution — CHANGES_REQUESTED** [review]

File: octoprint_mrbeam/camera/corners.py
Comment (khaledsherkawi) [reply octoprint_mrbeam/camera/corners.py:1]: There was 1 bug and 19 code smells; I resolved 1 bug and 17 code smells but somehow the result was 42 code smells

File: octoprint_mrbeam/static/js/app/view-models/working-area.js (hunk 1 above)
Comment (khaledsherkawi) [reply octoprint_mrbeam/static/js/app/view-models/working-area.js:629]: it is meaningless as beamos_vers can either be null or not so the ternary conditional operator is not needed

File: octoprint_mrbeam/camera/corners.py
Comment (khaledsherkawi) [reply octoprint_mrbeam/camera/corners.py:1]: The 42 code smells come from: octoprint_mrbeam/camera/undistort.py
I guess we can skip further refactoring for this PR

File: octoprint_mrbeam/templates/tab_workingarea.jinja2 (hunk 2 above)
Comment (khaledsherkawi) [reply octoprint_mrbeam/templates/tab_workingarea.jinja2]: done

**Review by JFSolution — DISMISSED** [review]

**Review by JFSolution — APPROVED** [review]

**Pull request merged**